flux/clusters/single-node/apps-variables.ini

```diff
@@ -0,0 +1,20 @@
+soperator_suspended=false
+slurm_cluster_suspended=false
+cert_manager_suspended=false
+monitoring_suspended=false
+
+# version of the soperator to install
+soperator_version=1.15.1-8b29c649
```

Comment:
May be we should add some version sync string to the Make file in order to update version here